src/main/scala/com/trueaccord/scalapb/json/JsonFormat.scala

```diff
@@ -32,6 +32,7 @@ case class FormatRegistry(mapClass: Map[Class[_], (_ => JValue, JValue => _)] =
 class Printer(
   includingDefaultValueFields: Boolean = false,
   preservingProtoFieldNames: Boolean = false,
+  formattingLongAsString: Boolean = false,
```

Comment:
Ok, I'll change this, I just was thinking this could break existing clients when they upgrade, but if you're ok with that then no problem :)